onnxscript/tests/function_libs/torch_lib/ops_test_common.py

```diff
@@ -276,10 +277,7 @@ def convert_tensor_to_numpy(input: Any) -> Any:
 
 
 def convert_kwargs_for_onnx(kwargs: dict[str, Any]) -> dict[str, Any]:
-    """Converts kwargs to be compatible with ONNX Runtime.
-
-    ONNX Runtime doesn't support torch.bool, so we convert them to torch.uint8.
```

Comment:
for my knowledge, does ort support torch.bool now? or was this docstring outdated already?

Reply:
I actually don't know if ORT supports bool (it should?), but I think this message was a mistake by copilot because we don't actually have this conversion logic. If we see issues with ORT I will make adjustments.